src/test/java/seedu/taassist/logic/parser/TaAssistParserTest.java

```diff
@@ -47,17 +65,66 @@ public void parseCommand_add() throws Exception {
 
     @Test
     public void parseCommand_addc() throws Exception {
```

Comment:
The original AB3 test cases throws Exception. However, we do not expect exceptions to be thrown when running the test cases. Perhaps we can consider handling the exceptions within the test methods. This is the same for other test cases. Perhaps we can consider handling this in the future commits.